tools/etc/frr/support_bundle_commands.conf

```diff
@@ -153,3 +153,41 @@ show ip pim state
 show ip pim statistics
 show ip pim rpf
 CMD_LIST_END
+
+# OSPFv3 Support Bundle Command List
+PROC_NAME:ospf6
+CMD_LIST_START
+show ipv6 ospf6
+do show ipv6 ospf6 border-routers
+do show ipv6 ospf6 border-routers detail
+show ipv6 ospf6 database
+show ipv6 ospf6 database detail
```

Comment:
we need both database and database detail?

Reply:
I have kept both considering that it is easier to first look into the database output and then detail if required.